includes/wc-update-functions.php

```diff
@@ -1200,6 +1205,7 @@ function wc_update_300_webhooks() {
 }
 
 /**
+>>>>>>> master
```

Comment:
Some unresolved merge conflicts in this file. This looks good to merge once those are fixed.

Reply:
Sorry, just fixed it.